src/google-classroom-extractor/edfi_google_classroom_extractor/mapping/assignments.py

```diff
@@ -68,13 +68,10 @@ def coursework_to_assignments_dfs(
                 "dueTime.minutes",
             ]
         ]
-        filled_df: Optional[DataFrame] = due_date_df.fillna("0")
```

Comment:
This was crashing on various oddities in the due date data from the live API during testing.